In [1]:
import numpy as np

## “Производная функции нескольких переменных. Часть 2

### 1. Исследовать функцию на условный экстремум

### $u=3-8x+6y$, если $x^{2} +y^{2} =36$

**Необходимое условие**

\begin{gather*}
L^{\prime }_{x} =\left( 3-8x+6y+\lambda \cdot \left( x^{2} +y^{2} -36\right)\right)^{\prime }_{x} =2x\lambda -8\\
L^{\prime }_{y} =\left( 3-8x+6y+\lambda \cdot \left( x^{2} +y^{2} -36\right)\right)^{\prime }_{y} =2y\lambda +6\\
\phi =x^{2} +y^{2} -36
\end{gather*}

\begin{gather*}
L=3-8x+6y+\lambda \cdot \left( x^{2} +y^{2} -36\right)
\end{gather*}

\begin{gather*}
\begin{cases}
L^{\prime }_{x} =0\\
L^{\prime }_{y} =0\\
\phi =0
\end{cases} \Rightarrow \begin{cases}
2x\lambda -8=0\\
2y\lambda +6=0\\
x^{2} +y^{2} -36=0
\end{cases}
\end{gather*}

**Стационарные точки:**

\begin{gather*}
2x\lambda -8=0\\
2x\lambda =8\\
x=\frac{8}{2\lambda } =\frac{4}{\lambda }\\
\\
2y\lambda +6=0\\
2y\lambda =-6\\
y=-\frac{6}{2\lambda } =-\frac{3}{\lambda }\\
\end{gather*}

\begin{gather*}
\left(\frac{4}{\lambda }\right)^{2} +\left( -\frac{3}{\lambda }\right)^{2} -36=0\\
\frac{16+9}{\lambda ^{2}} -36=0\\
\frac{25}{\lambda ^{2}} =36\\
\lambda =\pm \sqrt{\frac{25}{36}} =\pm \frac{5}{6}
\end{gather*}

\begin{gather*}
\lambda =-\frac{5}{6}\\
\begin{cases}
x=\frac{4}{\lambda }\\
y=-\frac{3}{\lambda }
\end{cases} \Rightarrow \begin{cases}
x=\frac{4}{-\frac{5}{6}} =4\cdot -\frac{6}{5} =-\frac{24}{5}\\
y=-\frac{3}{-\frac{5}{6}} =-3\cdot -\frac{5}{6} =\frac{18}{5}
\end{cases}\\
M_{1}\left( -\frac{24}{5} ;\ \frac{18}{5}\right)
\end{gather*}

\begin{gather*}
\lambda =\frac{5}{6}\\
\begin{cases}
x=\frac{4}{\lambda }\\
y=-\frac{3}{\lambda }
\end{cases} \Rightarrow \begin{cases}
x=\frac{4}{\frac{5}{6}} =4\cdot \frac{6}{5} =\frac{24}{5}\\
y=-\frac{3}{\frac{5}{6}} =-3\cdot \frac{5}{6} =-\frac{18}{5}
\end{cases}\\
M_{2}\left(\frac{24}{5} ;\ -\frac{18}{5}\right)
\end{gather*}

**Достаточное условие**

\begin{gather*}
\phi ^{\prime }_{x} =\left( x^{2} +y^{2} -36\right)^{\prime }_{x} =2x\\
\phi ^{\prime }_{y} =\left( x^{2} +y^{2} -36\right)^{\prime }_{y} =2y\\
\\
L^{''}_{xx} =( 2x\lambda -8)^{\prime }_{x} =2\lambda \\
L^{''}_{yy} =( 2y\lambda +6)^{\prime }_{y} =2\lambda \\
L^{''}_{xy} =( 2x\lambda -8)^{\prime }_{y} =0\\
L^{''}_{yx} =( 2y\lambda +6)^{\prime }_{x} =0\\
\end{gather*}

\begin{gather*}
A=\begin{bmatrix}
0 & \phi ^{\prime }_{x} & \phi ^{\prime }_{y}\\
\phi ^{\prime }_{x} & L^{''}_{xx} & L^{''}_{xy}\\
\phi ^{\prime }_{y} & L^{''}_{yx} & L^{''}_{yy}
\end{bmatrix} =\begin{bmatrix}
0 & 2x & 2y\\
2x & 2\lambda  & 0\\
2y & 0 & 2\lambda 
\end{bmatrix} =8\cdot \begin{bmatrix}
0 & x & y\\
x & \lambda  & 0\\
y & 0 & \lambda 
\end{bmatrix}
\end{gather*}

\begin{gather*}
\lambda =-\frac{5}{6} ;\ M_{1}\left( -\frac{24}{5} ;\ \frac{18}{5}\right)\\
A_{1} =8\cdot \begin{bmatrix}
0 & -\frac{24}{5} & \frac{18}{5}\\
-\frac{24}{5} & -\frac{5}{6} & 0\\
\frac{18}{5} & 0 & -\frac{5}{6}
\end{bmatrix}\\
| A_{1}| =8\cdot \left(\frac{24}{5} \cdot \begin{bmatrix}
-\frac{24}{5} & 0\\
\frac{18}{5} & -\frac{5}{6}
\end{bmatrix} +\frac{18}{5} \cdot \begin{bmatrix}
-\frac{24}{5} & -\frac{5}{6}\\
\frac{18}{5} & 0
\end{bmatrix}\right) =\\
=8\cdot \left(\frac{24}{5} \cdot \left( -\frac{24}{5} \cdot \left( -\frac{5}{6}\right)\right) +\frac{18}{5} \cdot \left( -\frac{24}{5} \cdot 0-\left( -\frac{5}{6} \cdot \frac{18}{5}\right)\right)\right) =\\
=8\cdot \left(\frac{96}{5} +\frac{54}{5}\right) =8\cdot \frac{150}{5} =8\cdot 30=240
\end{gather*}

\begin{gather*}
\lambda =\frac{5}{6} ;\ M_{2}\left(\frac{24}{5} ;\ -\frac{18}{5}\right)\\
A_{2} =8\cdot \begin{bmatrix}
0 & \frac{24}{5} & -\frac{18}{5}\\
\frac{24}{5} & \frac{5}{6} & 0\\
-\frac{18}{5} & 0 & \frac{5}{6}
\end{bmatrix}\\
| A_{2}| =8\cdot \left( -\frac{24}{5} \cdot \begin{bmatrix}
\frac{24}{5} & 0\\
-\frac{18}{5} & \frac{5}{6}
\end{bmatrix} -\frac{18}{5} \cdot \begin{bmatrix}
\frac{24}{5} & \frac{5}{6}\\
-\frac{18}{5} & 0
\end{bmatrix}\right) =\\
=8\cdot \left( -\frac{24}{5} \cdot \left(\frac{24}{5} \cdot \frac{5}{6}\right) -\frac{18}{5} \cdot \left(\frac{24}{5} \cdot 0-\left( -\frac{18}{5} \cdot \frac{5}{6}\right)\right)\right) =\\
=8\cdot \left( -\frac{96}{5} -\frac{54}{5}\right) =8\cdot \left( -\frac{150}{5}\right) =-8\cdot 30=-240
\end{gather*}

$| A_{1}| =240\rightarrow | A_{1}|  >0\rightarrow$ функция $u$ имеет максимум в точке $M_{1}\left( -\frac{24}{5} ;\ \frac{18}{5}\right)$


$| A_{2}| =-240\rightarrow | A_{1}| < 0\ \rightarrow$ функция $u$ имеет нимум в точке $M_{2}\left(\frac{24}{5} ;\ -\frac{18}{5}\right)$



\begin{gather*}
u_{max} =u\left( -\frac{24}{5} ;\ \frac{18}{5}\right) =3-8\cdot \left( -\frac{24}{5}\right) +6\cdot \frac{18}{5} =\\
=3+\frac{192}{5} +\frac{108}{5} =63\\
u_{min} =u\left(\frac{24}{5} ;\ -\frac{18}{5}\right) =3-8\cdot \frac{24}{5} +6\cdot \left( -\frac{18}{5}\right) =\\
=3-\frac{192}{5} -\frac{108}{5} =-57\\
\end{gather*}


### *Ответ:*

### **при условии** $x^{2} +y^{2} =36$

### $u_{max} =u\left( -\frac{24}{5} ;\ \frac{18}{5}\right) =63$

### $u_{min} =u\left(\frac{24}{5} ;\ -\frac{18}{5}\right) =-57$

### 2. Исследовать функцию на условный экстремум

### $u=2x^{2} +12xy+32y^{2} +15$, если $x^{2} +16y^{2} =64$

**Необходимое условие**

\begin{gather*}
L^{\prime }_{x} =\left( 2x^{2} +12xy+32y^{2} +15+\lambda \cdot \left( x^{2} +16y^{2} -64\right)\right)^{\prime }_{x} =4x+12y+2x\lambda \\
L^{\prime }_{y} =\left( 2x^{2} +12xy+32y^{2} +15+\lambda \cdot \left( x^{2} +16y^{2} -64\right)\right)^{\prime }_{y} =12x+64y+32y\lambda \\
\phi =x^{2} +16y^{2} -64
\end{gather*}

\begin{gather*}
L=2x^{2} +12xy+32y^{2} +15+\lambda \cdot \left( x^{2} +16y^{2} -64\right)
\end{gather*}

\begin{gather*}
\begin{cases}
L^{\prime }_{x} =0\\
L^{\prime }_{y} =0\\
\phi =0
\end{cases} \Rightarrow \begin{cases}
4x+12y+2x\lambda =0\\
12x+64y+32y\lambda =0\\
x^{2} +16y^{2} -64=0
\end{cases}
\end{gather*}

**Стационарные точки:**

\begin{gather*}
\frac{-4x-12y}{2x} =\frac{-16y-3x}{8y}\\
\frac{4y( -4x-12y)}{4xy} =\frac{x( -16y-3x)}{8xy}\\
\frac{-16xy-48y^{2}}{8xy} -\frac{-16xy-3x^{2}}{8xy} =0\\
\frac{-16xy-48y^{2} +16xy+3x^{2}}{8xy} =0\\
-48y^{2} +3x^{2} =0\\
\\
\begin{cases}
x^{2} +16y^{2} -64=0\\
3x^{2} -48y^{2} =0
\end{cases}\\
\\
x^{2} +16y^{2} -64=0\\
x^{2} =64-16y^{2}\\
x=\pm \sqrt{64-16y^{2}}\\
\end{gather*}

\begin{gather*}
3x^{2} -48y^{2} =0\\
3\left(\sqrt{64-16y^{2}}\right)^{2} -48y^{2} =0\\
3\left( 64-16y^{2}\right) -48y^{2} =0\\
192-48y^{2} -48y^{2} =0\\
192-96y^{2} =0\\
96y^{2} =192\\
y^{2} =\frac{192}{96} =2\\
y=\pm \sqrt{2}
\end{gather*}

\begin{gather*}
3x^{2} -48y^{2} =0\\
3x^{2} -48\left(\sqrt{2}\right)^{2} =0\\
3x^{2} -48\cdot 2=0\\
3x^{2} -96=0\\
x^{2} =\frac{96}{3} =32\\
x=\pm \sqrt{32} =\pm \sqrt{16\cdot 2} =\pm 4\sqrt{2}\\
x=\pm 4\sqrt{2}
\end{gather*}

\begin{gather*}
\lambda =\frac{-4x-12y}{2x} =\frac{-4\cdot 4\sqrt{2} -12\cdot \sqrt{2}}{2\cdot 4\sqrt{2}} =\frac{-4-3}{2} =-\frac{7}{2}\\
\\
\lambda =\frac{-4x-12y}{2x} =\frac{-4\cdot \left( -4\sqrt{2}\right) -12\cdot \left( -\sqrt{2}\right)}{2\cdot \left( -4\sqrt{2}\right)} =\frac{4+3}{-2} =-\frac{7}{2}\\
\end{gather*}

\begin{gather*}
\lambda =\frac{-4x-12y}{2x} =\frac{-4\cdot \left( -4\sqrt{2}\right) -12\cdot \sqrt{2}}{2\cdot \left( -4\sqrt{2}\right)} =\frac{-4+3}{2} =-\frac{1}{2}\\
\\
\lambda =\frac{-4x-12y}{2x} =\frac{-4\cdot 4\sqrt{2} -12\cdot \left( -\sqrt{2}\right)}{2\cdot 4\sqrt{2}} =\frac{-4+3}{2} =-\frac{1}{2}\\
\end{gather*}

\begin{gather*}
\lambda_{1} =-\frac{7}{2}\\
M_{1}\left( 4\sqrt{2} ;\sqrt{2}\right)\\
M_{2}\left( -4\sqrt{2} ;-\sqrt{2}\right)\\
M_{3}\left( -4\sqrt{2} ;\sqrt{2}\right)\\
M_{4}\left( 4\sqrt{2} ;-\sqrt{2}\right)
\end{gather*}

\begin{gather*}
\lambda_{2} =-\frac{1}{2}\\
M_{5}\left( 4\sqrt{2} ;\sqrt{2}\right)\\
M_{6}\left( -4\sqrt{2} ;-\sqrt{2}\right)\\
M_{7}\left( -4\sqrt{2} ;\sqrt{2}\right)\\
M_{8}\left( 4\sqrt{2} ;-\sqrt{2}\right)
\end{gather*}

**Достаточное условие**

\begin{gather*}
\phi ^{\prime }_{x} =\left( x^{2} +16y^{2} -64\right)^{\prime }_{x} =2x\\
\phi ^{\prime }_{y} =\left( x^{2} +16y^{2} -64\right)^{\prime }_{y} =32y\\
\\
L^{''}_{xx} =( 4x+12y+2x\lambda )^{\prime }_{x} =4+2\lambda \\
L^{''}_{yy} =( 12x+64y+32y\lambda )^{\prime }_{y} =64+32\lambda \\
L^{''}_{xy} =( 4x+12y+2x\lambda )^{\prime }_{y} =12\\
L^{''}_{yx} =( 12x+64y+32y\lambda )^{\prime }_{x} =12\\
\end{gather*}

\begin{gather*}
A=\begin{bmatrix}
0 & \phi ^{\prime }_{x} & \phi ^{\prime }_{y}\\
\phi ^{\prime }_{x} & L^{''}_{xx} & L^{''}_{xy}\\
\phi ^{\prime }_{y} & L^{''}_{yx} & L^{''}_{yy}
\end{bmatrix} =\begin{bmatrix}
0 & 2x & 32y\\
2x & 4+2\lambda  & 12\\
32y & 12 & 64+32\lambda 
\end{bmatrix} =2\cdot \begin{bmatrix}
0 & x & 16y\\
x & 2+\lambda  & 6\\
16y & 6 & 32+16\lambda 
\end{bmatrix}
\end{gather*}

\begin{gather*}
\lambda_{1} =-\frac{7}{2} ;\ M_{1}\left( 4\sqrt{2} ;\sqrt{2}\right)\\
A_{1} =2\cdot \begin{bmatrix}
0 & 4\sqrt{2} & 16\sqrt{2}\\
4\sqrt{2} & 2-\frac{7}{2} & 6\\
16\sqrt{2} & 6 & 32-16\cdot \frac{7}{2}
\end{bmatrix}
\end{gather*}

In [2]:
A1 = 2 * np.array([[0, 4 * np.sqrt(2), 16 * np.sqrt(2)],
                   [4 * np.sqrt(2), 2 - 7 / 2, 6],
                   [16 * np.sqrt(2), 6, 32 - 16 * 7 / 2]])
A1

array([[  0.       ,  11.3137085,  45.254834 ],
       [ 11.3137085,  -3.       ,  12.       ],
       [ 45.254834 ,  12.       , -48.       ]])

In [3]:
np.linalg.det(A1)

24576.000000000015

\begin{gather*}
| A_{1}| =24576
\end{gather*}

\begin{gather*}
\lambda_{1} =-\frac{7}{2} ;\ M_{2}\left( -4\sqrt{2} ;-\sqrt{2}\right)\\
A_{2} =2\cdot \begin{bmatrix}
0 & -4\sqrt{2} & -16\sqrt{2}\\
-4\sqrt{2} & 2-\frac{7}{2} & 6\\
-16\sqrt{2} & 6 & 32-16\cdot \frac{7}{2}
\end{bmatrix}
\end{gather*}

In [4]:
A2 = 2 * np.array([[0, -4 * np.sqrt(2), -16 * np.sqrt(2)],
                   [-4 * np.sqrt(2), 2 - 7 / 2, 6],
                   [-16 * np.sqrt(2), 6, 32 - 16 * 7 / 2]])
A2

array([[  0.       , -11.3137085, -45.254834 ],
       [-11.3137085,  -3.       ,  12.       ],
       [-45.254834 ,  12.       , -48.       ]])

In [5]:
np.linalg.det(A2)

24576.000000000015

\begin{gather*}
| A_{2}| =24576
\end{gather*}

\begin{gather*}
\lambda_{1} =-\frac{7}{2} ;\ M_{3}\left( -4\sqrt{2} ;\sqrt{2}\right)\\
A_{3} =2\cdot \begin{bmatrix}
0 & -4\sqrt{2} & 16\sqrt{2}\\
-4\sqrt{2} & 2-\frac{7}{2} & 6\\
16\sqrt{2} & 6 & 32-16\cdot \frac{7}{2}
\end{bmatrix}
\end{gather*}

In [6]:
A3 = 2 * np.array([[0, -4 * np.sqrt(2), 16 * np.sqrt(2)],
                   [-4 * np.sqrt(2), 2 - 7 / 2, 6],
                   [16 * np.sqrt(2), 6, 32 - 16 * 7 / 2]])
A3

array([[  0.       , -11.3137085,  45.254834 ],
       [-11.3137085,  -3.       ,  12.       ],
       [ 45.254834 ,  12.       , -48.       ]])

In [7]:
np.linalg.det(A3)

0.0

\begin{gather*}
| A_{3}| =0
\end{gather*}

\begin{gather*}
\lambda_{1} =-\frac{7}{2} ;\ M_{4}\left( 4\sqrt{2} ;-\sqrt{2}\right)\\
A_{4} =2\cdot \begin{bmatrix}
0 & 4\sqrt{2} & -16\sqrt{2}\\
4\sqrt{2} & 2-\frac{7}{2} & 6\\
-16\sqrt{2} & 6 & 32-16\cdot \frac{7}{2}
\end{bmatrix}
\end{gather*}

In [8]:
A4 = 2 * np.array([[0, 4 * np.sqrt(2), -16 * np.sqrt(2)],
                   [4 * np.sqrt(2), 2 - 7 / 2, 6],
                   [-16 * np.sqrt(2), 6, 32 - 16 * 7 / 2]])
A4

array([[  0.       ,  11.3137085, -45.254834 ],
       [ 11.3137085,  -3.       ,  12.       ],
       [-45.254834 ,  12.       , -48.       ]])

In [9]:
np.linalg.det(A4)

0.0

\begin{gather*}
| A_{4}| =0
\end{gather*}

\begin{gather*}
\lambda_{2} =-\frac{1}{2} ;\ M_{5}\left( 4\sqrt{2} ;\sqrt{2}\right)\\
A_{5} =2\cdot \begin{bmatrix}
0 & 4\sqrt{2} & 16\sqrt{2}\\
4\sqrt{2} & 2-\frac{1}{2} & 6\\
16\sqrt{2} & 6 & 32-16\cdot \frac{1}{2}
\end{bmatrix}
\end{gather*}

In [10]:
A5 = 2 * np.array([[0, 4 * np.sqrt(2), 16 * np.sqrt(2)],
                   [4 * np.sqrt(2), 2 - 1 / 2, 6],
                   [16 * np.sqrt(2), 6, 32 - 16 * 1 / 2]])
A5

array([[ 0.       , 11.3137085, 45.254834 ],
       [11.3137085,  3.       , 12.       ],
       [45.254834 , 12.       , 48.       ]])

In [11]:
np.linalg.det(A5)

0.0

\begin{gather*}
| A_{5}| =0
\end{gather*}

\begin{gather*}
\lambda_{2} =-\frac{1}{2} ;\ M_{6}\left( -4\sqrt{2} ;-\sqrt{2}\right)\\
A_{6} =2\cdot \begin{bmatrix}
0 & -4\sqrt{2} & -16\sqrt{2}\\
-4\sqrt{2} & 2-\frac{1}{2} & 6\\
-16\sqrt{2} & 6 & 32-16\cdot \frac{1}{2}
\end{bmatrix}
\end{gather*}

In [12]:
A6 = 2 * np.array([[0, -4 * np.sqrt(2), -16 * np.sqrt(2)],
                   [-4 * np.sqrt(2), 2 - 1 / 2, 6],
                   [-16 * np.sqrt(2), 6, 32 - 16 * 1 / 2]])
A6

array([[  0.       , -11.3137085, -45.254834 ],
       [-11.3137085,   3.       ,  12.       ],
       [-45.254834 ,  12.       ,  48.       ]])

In [13]:
np.linalg.det(A6)

0.0

\begin{gather*}
| A_{6}| =0
\end{gather*}

\begin{gather*}
\lambda_{2} =-\frac{1}{2} ;\ M_{7}\left( -4\sqrt{2} ;\sqrt{2}\right)\\
A_{7} =2\cdot \begin{bmatrix}
0 & -4\sqrt{2} & 16\sqrt{2}\\
-4\sqrt{2} & 2-\frac{1}{2} & 6\\
16\sqrt{2} & 6 & 32-16\cdot \frac{1}{2}
\end{bmatrix}
\end{gather*}

In [14]:
A7 = 2 * np.array([[0, -4 * np.sqrt(2), 16 * np.sqrt(2)],
                   [-4 * np.sqrt(2), 2 - 1 / 2, 6],
                   [16 * np.sqrt(2), 6, 32 - 16 * 1 / 2]])
A7

array([[  0.       , -11.3137085,  45.254834 ],
       [-11.3137085,   3.       ,  12.       ],
       [ 45.254834 ,  12.       ,  48.       ]])

In [15]:
np.linalg.det(A7)

-24576.000000000015

\begin{gather*}
| A_{7}| =-24576
\end{gather*}

\begin{gather*}
\lambda_{2} =-\frac{1}{2} ;\ M_{8}\left( 4\sqrt{2} ;-\sqrt{2}\right)\\
A_{8} =2\cdot \begin{bmatrix}
0 & 4\sqrt{2} & -16\sqrt{2}\\
4\sqrt{2} & 2-\frac{1}{2} & 6\\
-16\sqrt{2} & 6 & 32-16\cdot \frac{1}{2}
\end{bmatrix}
\end{gather*}

In [16]:
A8 = 2 * np.array([[0, 4 * np.sqrt(2), -16 * np.sqrt(2)],
                   [4 * np.sqrt(2), 2 - 1 / 2, 6],
                   [-16 * np.sqrt(2), 6, 32 - 16 * 1 / 2]])
A8

array([[  0.       ,  11.3137085, -45.254834 ],
       [ 11.3137085,   3.       ,  12.       ],
       [-45.254834 ,  12.       ,  48.       ]])

In [17]:
np.linalg.det(A8)

-24576.000000000015

\begin{gather*}
| A_{8}| =-24576
\end{gather*}

$| A_{1}| =24576\rightarrow | A_{1}|  >0\rightarrow$ функция $u$ имеет максимум в точке $M_{1}\left( 4\sqrt{2} ;\sqrt{2}\right)$


$| A_{2}| =24576\rightarrow | A_{2}| > 0\ \rightarrow$ функция $u$ имеет максимум в точке $M_{2}\left( -4\sqrt{2} ;-\sqrt{2}\right)$

$| A_{7}| =-24576\rightarrow | A_{7}|  < 0\rightarrow$ функция $u$ имеет нимум в точке $M_{7}\left( -4\sqrt{2} ;\sqrt{2}\right)$


$| A_{8}| =-24576\rightarrow | A_{8}| < 0\ \rightarrow$ функция $u$ имеет нимум в точке $M_{8}\left( 4\sqrt{2} ;-\sqrt{2}\right)$

\begin{gather*}
u_{max} =u\left( 4\sqrt{2} ;\sqrt{2}\right) =2\cdotp \left( 4\sqrt{2}\right)^{2} +12\cdot 4\sqrt{2} \cdot \sqrt{2} +32\cdot \left(\sqrt{2}\right)^{2} +15=\\
=2\cdotp 16\cdot 2+12\cdot 4\cdot 2+32\cdot 2+15=239\\
u_{max} =u\left( -4\sqrt{2} ;-\sqrt{2}\right) =2\cdotp \left( -4\sqrt{2}\right)^{2} +12\cdot \left( -4\sqrt{2}\right) \cdot \left( -\sqrt{2}\right) +32\cdot \left( -\sqrt{2}\right)^{2} +15=\\
=2\cdotp 16\cdot 2+12\cdot 4\cdot 2+32\cdot 2+15=239\\
u_{min} =u\left( -4\sqrt{2} ;\sqrt{2}\right) =2\cdotp \left( -4\sqrt{2}\right)^{2} +12\cdot \left( -4\sqrt{2}\right) \cdot \sqrt{2} +32\cdot \left(\sqrt{2}\right)^{2} +15=\\
=2\cdotp 16\cdot 2-12\cdot 4\cdot 2+32\cdot 2+15=47\\
u_{min} =u\left( 4\sqrt{2} ;-\sqrt{2}\right) =2\cdotp \left( 4\sqrt{2}\right)^{2} +12\cdot 4\sqrt{2} \cdot \left( -\sqrt{2}\right) +32\cdot \left( -\sqrt{2}\right)^{2} +15=\\
=2\cdotp 16\cdot 2-12\cdot 4\cdot 2+32\cdot 2+15=47
\end{gather*}

### *Ответ:*

### **при условии** $x^{2} +16y^{2} =64$

### $u_{max} =u\left( 4\sqrt{2} {;}\sqrt{2}\right) =239$

### $u_{max} =u\left( -4\sqrt{2} {;}-\sqrt{2}\right) =239$

### $u_{min} =u\left( -4\sqrt{2} {;}\sqrt{2}\right) =47$

### $u_{min} =u\left( 4\sqrt{2} {;}-\sqrt{2}\right) =47$

### 3. Найти производную функции $u=x^{2} +y^{2} +z^{2}$ по направлению вектора $\vec{c}( -9,8,-12)$ в точку $M( 8;-12;9)$

\begin{gather*}
u^{\prime }_{x} =\left( x^{2} +y^{2} +z^{2}\right)^{\prime }_{x} =2x\\
u^{\prime }_{y} =\left( x^{2} +y^{2} +z^{2}\right)^{\prime }_{y} =2y\\
u^{\prime }_{z} =\left( x^{2} +y^{2} +z^{2}\right)^{\prime }_{z} =2z
\end{gather*}

\begin{gather*}
u^{\prime }_{x}( M) =u^{\prime }_{x}( 8;-12;9) =2\cdot 8=16\\
u^{\prime }_{y}( M) =u^{\prime }_{y}( 8;-12;9) =2\cdot ( -12) =-24\\
u^{\prime }_{z}( M) =u^{\prime }_{z}( 8;-12;9) =2\cdot 9=18
\end{gather*}

\begin{gather*}
\overrightarrow{c_{0}} =\frac{\vec{c}}{\left| \vec{c}\right| } =\frac{-9\vec{i} +8\vec{j} -12\vec{k}}{\sqrt{( -9)^{2} +8^{2} +( -12)^{2}}} =\frac{-9\vec{i} +8\vec{j} -12\vec{k}}{\sqrt{289}} =\\
=-\frac{9}{17}\vec{i} +\frac{8}{17}\vec{j} -\frac{12}{17}\vec{k}
\end{gather*}

\begin{gather*}
\cos \alpha =-\frac{9}{17}\\
\cos \beta =\frac{8}{17}\\
\cos \gamma =-\frac{12}{17}
\end{gather*}

\begin{gather*}
\frac{\partial u}{\partial l} =u^{\prime }_{x}( M) \cdot \cos \alpha +\ u^{\prime }_{y}( M) \cdot \ \cos \beta +u^{\prime }_{z}( M) \cdot \cos \gamma \\
\frac{\partial u}{\partial l} =16\cdot \left( -\frac{9}{17}\right) -24\cdot \frac{8}{17} +18\cdot \left( -\frac{12}{17}\right) =\\
=-\frac{144}{17} -\frac{192}{17} -\frac{216}{17} =-\frac{552}{17}
\end{gather*}

### *Ответ: $\frac{\partial u}{\partial l} =-\frac{552}{17}$* 

### 4. Найти производную функции $e^{{x}^{2} +y^{2} +z^{2}}$ по направлению вектора $\vec{d}( 4,-13,-16)$ в точку $L( -16;4;-13)$

\begin{gather*}
u=e^{{x}^{2} +y^{2} +z^{2}}\\
\vec{d}( 4,-13,-16)\\
L( -16;4;-13)
\end{gather*}

\begin{gather*}
u^{\prime }_{x} =\left( e^{x^{2} +y^{2} +z^{2}}\right)^{\prime }_{x} =e^{x^{2} +y^{2} +z^{2}} \cdot 2x\\
u^{\prime }_{y} =\left( e^{x^{2} +y^{2} +z^{2}}\right)^{\prime }_{y} =e^{x^{2} +y^{2} +z^{2}} \cdot 2y\\
u^{\prime }_{z} =\left( e^{x^{2} +y^{2} +z^{2}}\right)^{\prime }_{z} =e^{x^{2} +y^{2} +z^{2}} \cdot 2z\\
\end{gather*}

\begin{gather*}
u^{\prime }_{x}( L) =u^{\prime }_{x}( -16;4;-13) =2\cdot ( -16) \cdot e^{( -16)^{2} +4^{2} +( -13)^{2}} =-32e^{441}\\
u^{\prime }_{y}( L) =u^{\prime }_{y}( -16;4;-13) =2\cdot 4\cdot e^{( -16)^{2} +4^{2} +( -13)^{2}} =8e^{441}\\
u^{\prime }_{z}( L) =u^{\prime }_{z}( -16;4;-13) =2\cdot ( -13) \cdot e^{( -16)^{2} +4^{2} +( -13)^{2}} =-26e^{441}
\end{gather*}

\begin{gather*}
\overrightarrow{d_{0}} =\frac{\vec{d}}{\left| \vec{d}\right| } =\frac{4\vec{i} -13\vec{j} -16\vec{k}}{\sqrt{4^{2} +( -13)^{2} +( -12)^{2}}} =\frac{4\vec{i} -13\vec{j} -16\vec{k}}{\sqrt{441}} =\\
=\frac{4}{21}\vec{i} -\frac{13}{21}\vec{j} -\frac{16}{21}\vec{k}\\
\\
\cos \alpha =\frac{4}{21}\\
\cos \beta =-\frac{13}{21}\\
\cos \gamma =-\frac{16}{21}
\end{gather*}

\begin{gather*}
\frac{\partial u}{\partial l} =u^{\prime }_{x}( L) \cdot \cos \alpha +\ u^{\prime }_{y}( L) \cdot \ \cos \beta +u^{\prime }_{z}( L) \cdot \cos \gamma \\
\frac{\partial u}{\partial l} =-32e^{441} \cdot \frac{4}{21} +8e^{441} \cdot \left( -\frac{13}{21}\right) -26e^{441} \cdot \left( -\frac{16}{21}\right) =\\
=-\frac{128e^{441}}{21} -\frac{104e^{441}}{21} +\frac{416e^{441}}{21} =\\
=\frac{e^{441} \cdot ( -128-104+416)}{21} =\frac{184e^{441}}{21}
\end{gather*}

### *Ответ: $\frac{\partial u}{\partial l} =\frac{184e^{441}}{21}$* 